frontend/public/locales/en/common.json

```diff
@@ -100,6 +100,7 @@
   "rss-description": "Subscribe to RSS feeds from Flathub",
   "rss-applications": "Do you need an RSS application? Find excellent ones on <1>Flathub</1>.",
   "applications-by-developer": "Applications by {{developer}}",
+  "applications-by-projectgroup": "Applications in the {{projectgroup}} group",
```

Comment:
I kinda want to use the same wording as for developer here, I'm not convinced the `group` part is needed

Reply:
Having played with this a bit, I don't think we can drop the group wording. As flatpak and gnome for e.g. have some entries, that I wouldn't expect and having the wording not use group, makes it sound too official in my mind.